### PR TITLE
[UseDotNetV2]fix to handle dotnet release preview channels

### PR DESCRIPTION
**Pull request by tejasd1990**
Fix for https://github.com/microsoft/azure-pipelines-tasks/issues/11569

### CODE REVIEW

**Comment by bishal-pdMSFT** [comment]
@hiyadav please review this

File: Tasks/UseDotNetV2/Strings/resources.resjson/en-US/resources.resjson [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "loc.friendlyName": "Use .NET Core",
-  "loc.helpMarkDown": "[Learn more about this task](https://aka.ms/AA4xgy0)",
```
Comment (hiyadav) [inline Tasks/UseDotNetV2/Strings/resources.resjson/en-US/resources.resjson]: We should revert back this change, i'll update the shortlink to point to the docs url

**Comment by hiyadav** [comment]
@tejasd1990 PR looks good to me. Just revert the readme link, I'll update the short URL to point to the microsoft doc for task.

File: Tasks/UseDotNetV2/Strings/resources.resjson/en-US/resources.resjson (hunk 1 above)
Comment (tejasd1990) [reply Tasks/UseDotNetV2/Strings/resources.resjson/en-US/resources.resjson]: corrected in task.json, the generated files got autocorrected.

**Pull request merged**